src/Mod/Draft/draftguitools/gui_grid.py

```diff
@@ -55,14 +56,18 @@ def __init__(self):
     def GetResources(self):
         """Set icon, menu and tooltip."""
 
-        return {"Pixmap": "Draft_Grid",
-                "Accel": "G, R",
-                "MenuText": QT_TRANSLATE_NOOP("Draft_ToggleGrid", "Toggle grid"),
-                "ToolTip": QT_TRANSLATE_NOOP("Draft_ToggleGrid",
-                                             "Toggles the Draft grid on and off."),
-                "CmdType": "ForEdit"}
+        return {
+            "Pixmap": "Draft_Grid",
+            "Accel": "G, R",
+            "CmdType": "ForEdit",
+            "MenuText": QT_TRANSLATE_NOOP("Draft_ToggleGrid", "Toggle grid"),
+            "ToolTip": QT_TRANSLATE_NOOP(
+                "Draft_ToggleGrid", "Toggles the Draft grid on and off."
+            ),
+            'Checkable': False,
```

Comment:
Use double quotes please. I know that the use single and double quotes is not consistent in the Draft WB. But I happen to have fixed that for this file in the current dev cycle.

Reply:
Happy to, I had noticed the inconsistency. I wanted to check first if `Checkable` was needed at all, so I might remove it in the end.

Reply:
```suggestion
            "Checkable": False,
```